spec.bs

```diff
@@ -1159,12 +1377,202 @@ fields:
 
     :  <dfn>instrument</dfn> member
     :: The instrument information that was displayed to the user.
+
+    :  <dfn>browserBoundPublicKey</dfn> member
+    :: The base64url encoding of the [=browser bound key=] public key. See 
 </dl>
 
 Note that there is no `paymentRequestOrigin` field in
 {{CollectedClientAdditionalPaymentData}}, because the origin of the calling
 frame is already included in {{CollectedClientData}} of [[webauthn-3]].
 
+## <dfn dictionary>CollectedClientAdditionalPaymentRegistrationData</dfn> Dictionary ## {#sctn-collectedclientadditionalpaymentregistrationdata-dictionary}
+
+<xmp class="idl">
+    dictionary CollectedClientAdditionalPaymentRegistrationData {
+        USVString browserBoundPublicKey;
+    };
+</xmp>
+
+The {{CollectedClientAdditionalPaymentRegistrationData}} dictionary contains the following
+fields:
+
+<dl dfn-type="dict-member" dfn-for="CollectedClientAdditionalPaymentRegistrationData">
+    :  <dfn>browserBoundPublicKey</dfn> member
+    :: The base64url encoding of the [=browser bound key=] public key.
+</dl>
+
+# Browser Bound Key Store - <dfn interface>BrowserBoundKeyStore</dfn> Internal Component # {#sctn-browser-bound-key-store}
+
+This component in the user agent manages platform-dependant cryptographic
+key pairs including their association to SPC credentials (i.e. passkeys). This
+section explains the {{BrowserBoundKeyStore}} and procedures for creating,
+binding, retrieving, and signing using the browser bound key pairs.
+
+Note: The BrowserBoundKeyStore needs to generate key pairs, retrieve key pairs,
+    export public keys, and generate signatures. Many operating system provide
+    APIs implement these operations and provide storage of the private in a
+    secure element on the device when available. For example [Android
+    KeyStore](https://source.android.com/docs/security/features/keystore),
+    [Apple CryptoKit](https://developer.apple.com/documentation/cryptokit/),
+    [Windows Cryptography API: Next
+    Generation](https://learn.microsoft.com/en-us/windows/win32/seccng/cng-portal).
+
+The browser bound key store contains
+
+    : |browser_bound_map|
+    :: a map from [[=Credential ID=]] to a key pair identifier.
+
+    : |keypair_map|
+    :: a map from key pair identifier to the public private key pair.
+
+        Note: The user agent may be using a cryptographic API that provides an
+            equivalent of the |keypair_map| in which case the user agent would
+            delegate this aspect to the cryptographic API.
+
+## Creating a KeyPair Internal Method ## {#sctn-creating-a-keypair}
+
+<dfn for="BrowserBoundKeyStore" method>CreateKeyPair(allowed_algorithms)</dfn>:
+
+Given |allowed_algorithms| a {{PublicKeyCredentialParameters}} object from
+{{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}, creates a
```

Comment:
I don't think you need to specify its from a {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}. Just something like:

`Given a {{PublicKeyCredentialParameters}}, |allowed_algorithms|, creates a ...`